frontend/src/utils/strategyNames.tsx

```diff
@@ -6,7 +6,7 @@ import LanguageIcon from '@mui/icons-material/Language';
 import PowerSettingsNewIcon from '@mui/icons-material/PowerSettingsNew';
 import CodeIcon from '@mui/icons-material/Code';
 import { ReactComponent as RolloutIcon } from 'assets/icons/rollout.svg';
-import FormatListNumberedIcon from '@mui/icons-material/FormatListNumbered';
+import FactCheckOutlinedIcon from '@mui/icons-material/FactCheckOutlined';
```

Comment:
Do we prefer the original name here or assigning as the name of what it's being used as?